sys-power/throttled/throttled-0.10.0.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit python-single-r1 systemd
+
+DESCRIPTION="Workaround for Intel throttling issues in Linux."
```

Comment:
I know the description is copied from original website, but it sounds a little bit weird to me.. if you can't think better improvement, then I'd be just fine ..

Reply:
Let's drop the full stop at the end either way, but if we can come up with something more descriptive, great